docs/secondary-navigation.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: sub-navigation
+order: 5
+title: Secondary navigation
+description: Allow users to navigate between pages scoped to a particular section.
+---
+
+{% from "example/macro.njk" import appExample %}
+
+{{ appExample({
+  component: "secondary-navigation"
+}) }}
+
+## When to use this component
+
+Use the secondary navigation when you already have a [primary navigation](primary-navigation) and need to let users navigate between pages that are scoped to a particular section or concept.
+
+For example, if your service is a case management system, you might use the secondary navigation if there are multiple pages associated with a single case.
+
+{{ appExample({
+  component: "secondary-navigation",
+  example: "with-header"
+}) }}
+
+## When not to use this component
+
+Do not use this component for the primary sections of your service, or if a [sub-navigation](sub-navigation) would be more appropriate.
+
+You could also consider using the [tabs component](https://design-system.service.gov.uk/components/tabs/) if the amount of content in each sub-section is small.
```

Comment:
Do we want to add something about how tabs should be used client-side, and secondary navigation should be used server-side, if that makes sense?

Maybe unnecessary as this is the default behaviour, but I’ve seen cases in the past where the tab component has been ‘hijacked’ to link to different pages, whereas this component should be used in that instance instead.

Reply:
I'm not sure we need to use the words client-side or server-side, but maybe we could go into a bit more detail about the difference between navigation and tabs, and when to use (and not use) either one?

The Tabs page goes into a fair bit of detail, including this:

> Do not use the tabs component if the total amount of content the tabs contain will make the page slow to load. For this reason, do not use the tabs component as a form of page navigation.

But yeah, I’ve also seen this ignored and the Tabs styled re-used as navigation instead (which is probably good evidence for the need for secondary navigation?).

Reply:
Wasn’t suggesting actually saying client-side/sever-side 😉 Perhaps something we can ignore until we see a need to add additional guidance?

Reply:
Update the guidance - take another look and let me know what you think. I added a "How it works" section, following the convention from the GOV.UK Design System website.